src/app/applications/components/application-parameters/kustomize.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { FieldApi, FormField as ReactFormField } from 'react-form';
+import { Checkbox } from '../../../../../node_modules/argo-ui';
+
+export interface Image {
+    name: string;
+    newName?: string;
+    newTag?: string;
+    digest?: string;
+}
+
+const pattern = /^(.*):([a-zA-Z0-9._-]*)$/;
+
+function parseOverwrite(arg: string, overwriteImage: boolean): { name: string; digest?: string, tag?: string } {
```

Comment:
This is BL in the FE. Perhaps it's time for `yarn test`?

Reply:
I agree. Lets add first unit tests. Working on it

Reply:
done: https://github.com/argoproj/argo-cd-ui/pull/125/files#diff-513b5bbde593665bce8f6bf307851b8c